### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by YukioRivera**

### CODE REVIEW

File: cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java
Comment (jennahyasin) [inline cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java:1]: Code looks clean and organized. There is commented out code that you can delete on lines 98-101. 
You might want to add a few comments for documenting your code.

File: cst438_gradebook/src/main/java/com/cst438/domain/AssignmentDTO.java
Comment (jennahyasin) [inline cst438_gradebook/src/main/java/com/cst438/domain/AssignmentDTO.java:1]: Making all the variables public was a good call to make it easier to access these variables in different classes. Looks good.

File: cst438_gradebook/src/test/java/com/cst438/JunitTestAssignment.java
Comment (jennahyasin) [inline cst438_gradebook/src/test/java/com/cst438/JunitTestAssignment.java:1]: Great job on creating JUnit tests for all three functions in ur AssignmentController class. 
There are a few different areas with commented out code that could be deleted. You also should add some more comments explaining ur reasoning for testing.

**Review by jennahyasin — COMMENTED** [review]
I reviewed all of the files in the dev that have been updated from the master branch. I made specific comments on the files individually, but overall, your code looks good. I had minimal suggestions of cleaning up some of your code by removing commented out codes and also adding some comments for clarifying your code (documentation). Great work!

**Comment by YukioRivera** [comment]
Thanks for the comments, I'll update the changes you suggested and work on the improves before the next commit.

**Pull request merged**